include/sentry.h

```diff
@@ -185,8 +187,6 @@ typedef enum {
     SENTRY_VALUE_TYPE_NULL,
     SENTRY_VALUE_TYPE_BOOL,
     SENTRY_VALUE_TYPE_INT32,
-    SENTRY_VALUE_TYPE_INT64,
-    SENTRY_VALUE_TYPE_UINT64,
```

Comment:
Why?

Reply:
that's an accident, I've done `git checkout master ./include/sentry.h` yesterday when restoring the sdk name change but looks like I didn't have latest master fetched.